src/formatDate.js

```diff
@@ -50,7 +50,41 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateParts = date.split(fromFormat[3]);
+
+  const dateObj = {};
+
+  for (let i = 0; i < 3; i++) {
+    if (fromFormat[i] === 'YY' || fromFormat[i] === 'YYYY') {
+      dateObj.year = dateParts[i];
+    } else if (fromFormat[i] === 'MM') {
+      dateObj.month = dateParts[i];
+    } else if (fromFormat[i] === 'DD') {
+      dateObj.day = dateParts[i];
+    }
```

Comment:
```suggestion
    } 
      dateObj[fromFormat[i]] = dateParts[i];
```
You can just use MM and DD instead of month and day